NDB.Covid19/NDB.Covid19.iOS/AppDelegate.cs

```diff
@@ -201,6 +205,17 @@ public void PerformFetch(UIApplication application, System.Action<UIBackgroundFe
         {
             completionHandler(UIBackgroundFetchResult.NewData);
         }
+
+        [Export("applicationWillTerminate:")]
+        public void AppWillTerminate(UIApplication application)
```

Comment:
Did you manage to trigger this? @MIKKH

Reply:
@pavshr Yes - I tried closing the app by swiping it away from active app's at different places during the submission flow, and it triggered the log message.